Sources/FunctionalTableData/CellActions.swift

```diff
@@ -72,7 +72,7 @@ public struct CellActions {
 				return rowAction
 			}
 			
-			@available(iOSApplicationExtension 11.0, *)
```

Comment:
Wonder why it was this way.

Reply:
Unsure as well, maybe a blanket replace to match our other frameworks. But Xcode 11 wasn’t having ANY of that now 😂